Automatic translation CoC (#890)
ropensci/roweb3: content/code-of-conduct.pt.md

\+++
title = "Código de Conduta (versão em inglês)"
description = "Código de Conduta da rOpenSci"
subtítulo = "Versão 2.5 - 30 de janeiro de 2024"
\+++

*O código de conduta também está disponível em [espanhol](/es/codigo-de-conducta).*

> A comunidade da rOpenSci é o nosso melhor patrimônio e acreditamos que a nossa diversidade é a nossa força. Estamos construindo uma comunidade global acolhedora e diversificada de usuários e desenvolvedores de software de vários domínios de pesquisa. Isso é tão importante para nós que está em [nossa declaração de missão](/about/). Seja você um colaborador assíduo ou um recém-chegado, nós nos preocupamos em fazer deste um lugar seguro para você e estamos ao seu lado.

Reviewer: ```suggestion
A comunidade da rOpenSci é o nosso melhor patrimônio e acreditamos que a nossa diversidade é a nossa força. Estamos construindo uma comunidade global acolhedora e diversificada de pessoas usuárias e desenvolvedoras de software de vários domínios de pesquisa. Isso é tão importante para nós que está em [nossa declaração de missão](/about/). Seja você uma pessoa colaboradora assídua ou um recém-chegada, nós nos preocupamos em fazer deste um lugar seguro para você e estamos ao seu lado.
```

> A rOpenSci tem o compromisso de oferecer uma experiência segura, inclusiva, acolhedora e livre de assédio para todos.

Reviewer: ```suggestion
A rOpenSci tem o compromisso de oferecer uma experiência segura, inclusiva, acolhedora e livre de assédio para todas as pessoas.
```

Tentamos cultivar uma comunidade com valores compartilhados, onde as pessoas se sintam à vontade para explorar ideias, fazer perguntas e dizer coisas como "Não entendo" ou "Por quê". Presuma a competência das pessoas com quem você interage. Não há perguntas idiotas.

> Seja atencioso ao falar e agir, e procure ativamente reconhecer e respeitar os limites das pessoas que participam das atividades do rOpenSci.

Reviewer: ```suggestion
Seja atencioso(a) ao falar e agir, e procure ativamente reconhecer e respeitar os limites das pessoas que participam das atividades da rOpenSci.
```

> Cuidem uns dos outros. Alerte um membro do Comitê do Código de Conduta (nomeado abaixo) se você notar uma situação perigosa, alguém em perigo ou uma possível violação deste Código de Conduta, mesmo que pareça inconsequente.

Reviewer: ```suggestion
Cuidem uns dos outros. Alerte um(a) membro(a) do Comitê do Código de Conduta (nomeado abaixo) se você notar uma situação perigosa, alguém em perigo ou uma possível violação deste Código de Conduta, mesmo que pareça inconsequente.
```
"Cuidem uns dos outros": pensar

Reviewer 2: ```suggestion
Seja atencioso ao falar e agir, e procure ativamente reconhecer e respeitar os limites das pessoas que participam das atividades da rOpenSci.
```

Reviewer 2: "Cuidemos umas das outras pessoas", fica estranho né?!

Author: Thank you for your review. I have only this comment left because I don't understand if I should change the text to @fblpalmeira's suggestion or leave the original one. If you let me know, we have our first version in Portuguese!!! cc: @beatrizmilz

Reviewer: > "Cuidemos umas das outras pessoas", fica estranho né?!

Fica um pouco, mas não consigo pensar em outra forma 😢 

Eu até tentei perguntar para o Chat GPT e todas as sugestões eram ruins..

Author: Here some more: 
- Vamos cuidar de nós mesmos
- Vamos nos cuidar
These are Deepl suggestions.

Author: Also, we can change the words to try to say the same.  What about: 
- Cuidando de nossa comunidade
- Cuidar da nossa comunidade
- Cuidar de outras pessoas na comunidade.

Reviewer: I think "Cuidar de outras pessoas na comunidade." is nice, gives the message that we need, and is gender neutral!

@fblpalmeira what do you think?

```suggestion
Vamos cuidar das outras pessoas na comunidade. Alerte um membro do Comitê do Código de Conduta (nomeado abaixo) se você notar uma situação perigosa, alguém em perigo ou uma possível violação deste Código de Conduta, mesmo que pareça inconsequente.
```

Reviewer 2: Perfeito, ficou ótimo!

- padrão de contato social inadequado, como solicitar/assumir níveis inadequados de intimidade com outras pessoas
- contato físico inadequado sem consentimento ou após uma solicitação para parar. Em ambientes virtuais, contato físico simulado inadequado usado sem consentimento, como descrições textuais como "abraço" ou "massagem nas costas" ou emojis que representam esses contatos.
- Perseguição ou acompanhamento, incluindo, em ambientes virtuais, comportamentos como mensagens diretas indesejadas persistentes, envio de fotos indesejadas ou links de malware, ou registro de atividades on-line para fins de assédio.
- Ameaças de violência ou incitação à violência contra qualquer indivíduo, inclusive incentivar uma pessoa a cometer suicídio ou a se automutilar
- uso de imagens, comentários ou piadas de cunho sexual ou discriminatório
- atenção sexual indesejada, que pode incluir perguntas inadequadas de natureza sexual, solicitação de favores sexuais ou pedidos repetidos de encontros ou informações de contato.
- "Divulgação" deliberada de qualquer aspecto da identidade de uma pessoa sem o seu consentimento, exceto quando necessário para proteger pessoas vulneráveis de abuso intencional

> - uso deliberado de gênero incorreto ou de nomes "mortos" ou rejeitados

Reviewer: Como falamos sobre o "dead name" em português? Pesquisar.

Reviewer 2: Acho que pode ser usado "nome civil" para "dead name".

> - uso deliberado de gênero incorreto ou de nomes "mortos" ou rejeitados

Author: ```suggestion
- uso deliberado de gênero incorreto ou de nome civil ou rejeitados
```

- publicação não consensual de comunicação privada não ofensiva, incluindo capturas de tela de canais de discussão privados ou mensagens diretas.

Priorizamos a segurança das pessoas marginalizadas em detrimento do conforto das pessoas privilegiadas. Não daremos seguimento a reclamações relacionadas a:

> - "reversos" -ismos, incluindo "racismo reverso", "sexismo reverso" e "cisfobia

Reviewer: ```suggestion
- "reversos" -ismos, incluindo "racismo reverso", "sexismo reverso" e "cisfobia"
```

- comunicação razoável de limites, como "deixe-me em paz", "vá embora" ou "não vou discutir isso com você"
- comunicação em um "tom" que você não considera agradável
- criticar comportamentos ou suposições racistas, sexistas, cissexistas ou opressivas de outra forma

Se alguém se envolver em comportamento de assédio, o Comitê tomará medidas que podem incluir avisar o infrator ou pedir que ele deixe um evento ou um canal on-line temporária ou permanentemente, ou que procure ajuda das autoridades policiais.

> Este Código de Conduta aplica-se a todas as pessoas que participam da comunidade rOpenSci, incluindo a equipe e a liderança da rOpenSci. Aplica-se a todos os modos de interação on-line, incluindo os repositórios de projetos do GitHub, o fórum de discussão do rOpenSci, o Slack, as chamadas da comunidade e pessoalmente em eventos organizados pelo rOpenSci ou eventos oficialmente endossados pelo rOpenSci, incluindo reuniões sociais afiliadas ao evento.

Reviewer: ```suggestion
Este Código de Conduta aplica-se a todas as pessoas que participam da comunidade rOpenSci, incluindo a equipe e a liderança da rOpenSci. Aplica-se a todos os modos de interação on-line, incluindo os repositórios de projetos do GitHub, o fórum de discussão da rOpenSci, o Slack, as chamadas da comunidade e pessoalmente em eventos organizados pela rOpenSci ou eventos oficialmente endossados pela rOpenSci, incluindo reuniões sociais afiliadas ao evento.
```

### Diretrizes para relatórios

> O Comitê do Código de Conduta da rOpenSci inclui [Natalia Morandeira](https://nmorandeira.netlify.app) (membro independente da comunidade), [Mark Padgham](/author/mark-padgham) (cientista pesquisador de software da rOpenSci) e [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) (Gerente da Comunidade rOpenSci).

Reviewer: ```suggestion
O Comitê do Código de Conduta da rOpenSci inclui [Natalia Morandeira](https://nmorandeira.netlify.app) (membra independente da comunidade), [Mark Padgham](/author/mark-padgham) (cientista pesquisador de software da rOpenSci) e [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) (Gerente da Comunidade rOpenSci).
```

Membra soa estranho, mas pelo que pesquisei, é a forma correta para o feminino de membro.

Reviewer 2: E ela é membro? Parece que a palavra membro é uma palavra sobrecomum e pode ser usada para os dois gêneros. Membra também é correto, mas menos comum.

Author: In Spanish miembro is used by masculine and feminine.  We can keep membro here.

> Se você presenciar ou testemunhar um comportamento inaceitável, ou tiver qualquer outra preocupação, [envie um relatório ao Comitê o mais rápido possível usando este formulário](https://forms.gle/8zKztThp2yLJ6KYa6). Como alternativa, você pode enviar um e-mail para `conduct@ropensci.org` que será enviado aos membros do Comitê. Você também pode fazer uma denúncia diretamente a um membro do Comitê entrando em contato com ele diretamente. Se algum membro do Comitê tiver um conflito de interesses com um relatório, ele será excluído e não terá acesso ao conteúdo ou ao processo de acompanhamento do relatório. Em um evento rOpenSci presencial, entre em contato com qualquer pessoa nomeada como membro do Comitê do Código de Conduta específico do evento, ou com a equipe ou segurança do local que estará presente durante todo o evento.

Reviewer: ```suggestion
Se você presenciar ou testemunhar um comportamento inaceitável, ou tiver qualquer outra preocupação, [envie um relatório ao Comitê o mais rápido possível usando este formulário](https://forms.gle/8zKztThp2yLJ6KYa6). Como alternativa, você pode enviar um e-mail para `conduct@ropensci.org` que será enviado aos(as) membros(as) do Comitê. Você também pode fazer uma denúncia diretamente a um(a) membro(a) do Comitê entrando em contato com ele(a) diretamente. Se algum(a) membro(a) do Comitê tiver um conflito de interesses com um relatório, ele(a) será excluído(a) e não terá acesso ao conteúdo ou ao processo de acompanhamento do relatório. Em um evento rOpenSci presencial, entre em contato com qualquer pessoa nomeada como membro do Comitê do Código de Conduta específico do evento, ou com a equipe ou segurança do local que estará presente durante todo o evento.
```

Author: Some suggestion:
- "que será enviado aos(as) membros(as) do Comitê" can be "que será enviado ao comitê"
- "Se algum(a) membro(a) do Comitê tiver um" can be "Se alguém do comitê tiver um"
- "ele(a) será excluído(a)" can be "essa pessoa será excluída"

If you agree, please edit your review so we can then apply the suggestions. Thanks!!

> Se você estiver planejando participar de um próximo evento ou entrar em um canal on-line e tiver preocupações com relação a outra pessoa que possa estar presente, envie uma denúncia conforme descrito acima. Trabalharemos com você para tomar as precauções necessárias para garantir seu conforto e segurança. Essas precauções podem incluir: fornecer uma escolta, preparar a equipe do evento no local e/ou fornecer números de telefone celular de contato no local para contato imediato. Em alguns casos, podemos tomar medidas para impedir que o assediador participe da conferência.

Reviewer: ```suggestion
Se você estiver planejando participar de um próximo evento ou entrar em um canal on-line e tiver preocupações com relação a outra pessoa que possa estar presente, envie uma denúncia conforme descrito acima. Trabalharemos com você para tomar as precauções necessárias para garantir seu conforto e segurança. Essas precauções podem incluir: fornecer uma escolta, preparar a equipe do evento no local e/ou fornecer números de telefone celular de contato no local para contato imediato. Em alguns casos, podemos tomar medidas para impedir que a pessoa assediadora participe da conferência.
```

> O Comitê do Código de Conduta investigará e decidirá as respostas às denúncias com o objetivo de tomar uma decisão e implementar a aplicação da lei assim que for razoavelmente possível. Estamos comprometidos com a transparência com nossa comunidade e, ao mesmo tempo, com a privacidade das vítimas e das pessoas que relatam incidentes. Publicaremos resumos breves e anônimos dos incidentes no site [fórum](https://discuss.ropensci.org/c/conduct) em uma categoria dedicada, somente para leitura, e publicaremos no blog do rOpenSci um relatório anual de transparência.

Reviewer: ```suggestion
O Comitê do Código de Conduta investigará e decidirá as respostas às denúncias com o objetivo de tomar uma decisão e implementar a aplicação da lei assim que for razoavelmente possível. Estamos comprometidos com a transparência com nossa comunidade e, ao mesmo tempo, com a privacidade das vítimas e das pessoas que relatam incidentes. Publicaremos resumos breves e anônimos dos incidentes no site [fórum](https://discuss.ropensci.org/c/conduct) em uma categoria dedicada, somente para leitura, e publicaremos no blog da rOpenSci um relatório anual de transparência.
```

> Agradecemos seus comentários por e-mail para `conduct@ropensci.org` e agradecemos a você por trabalhar conosco para tornar a rOpenSci uma experiência segura, agradável, amigável e enriquecedora para todos que participam.

Reviewer: ```suggestion
Agradecemos seus comentários por e-mail para `conduct@ropensci.org` e agradecemos a você por trabalhar conosco para tornar a rOpenSci uma experiência segura, agradável, amigável e enriquecedora para todas as pessoas participantes.
```

> #### Ex-alunos

Reviewer: ```suggestion
#### Ex-participantes do Comitê
```

Ex-alunos é algo estranho...

Agradecemos a Scott Chamberlain, Stefanie Butland, Megan Carter e Kara Woo por seus serviços anteriores no comitê.

#### Licença e créditos

> O Código de Conduta da rOpenSci foi influenciado e adaptado de muitas fontes, incluindo [Conversa sobre código aberto e sentimentos](https://youtu.be/nizfHxg8y3o) de [Audrey Eschright](https://lifeofaudrey.com/) Você está lendo o rascunho do Grupo de Trabalho de Diversidade e Inclusão da Comunidade do Consórcio R [Código de Conduta](https://github.com/RConsortium/RCDI-WG/blob/0ca0a91dccc9296ff53a5806f52a2a49dbb8850d/conduct/code-of-conduct.md) o [Política antiassédio do Geek Feminism](https://geekfeminism.wikia.com/wiki/Community_anti-harassment/Policy) a [Código de Conduta da useR! 2021](https://user2021.r-project.org/participation/coc/) e nossa própria Chamada da Comunidade: [Como faço para criar um Código de Conduta para meu evento/laboratório/codebase?](/commcalls/2016-12-15/)

Reviewer: ```suggestion
O Código de Conduta da rOpenSci foi influenciado e adaptado de muitas fontes, incluindo [Conversa sobre código aberto e sentimentos](https://youtu.be/nizfHxg8y3o) de [Audrey Eschright](https://lifeofaudrey.com/) Você está lendo o rascunho do Grupo de Trabalho de Diversidade e Inclusão da Comunidade da *R Consortium* [Código de Conduta](https://github.com/RConsortium/RCDI-WG/blob/0ca0a91dccc9296ff53a5806f52a2a49dbb8850d/conduct/code-of-conduct.md) o [Política antiassédio do Geek Feminism](https://geekfeminism.wikia.com/wiki/Community_anti-harassment/Policy) a [Código de Conduta da useR! 2021](https://user2021.r-project.org/participation/coc/) e nossa própria Chamada da Comunidade: [Como faço para criar um Código de Conduta para meu evento/laboratório/codebase?](/commcalls/2016-12-15/)
```

<p xmlns:dct="http://purl.org/dc/terms/" xmlns:vcard="http://www.w3.org/2001/vcard-rdf/3.0#">
  <a rel="license"
     href="https://creativecommons.org/publicdomain/zero/1.0/">
    <img src="https://i.creativecommons.org/p/zero/1.0/88x31.png" style="border-style: none; width: 88px;" alt="CC0" />
  </a>
  <br />

>   Na medida do possível, de acordo com a lei,

Reviewer: ```suggestion
  Na medida do possível, de acordo com a lei, a
```

  <a rel="dct:publisher"
     href="https://ropensci.org/">
    <span property="dct:title">rOpenSci</span></a>

>   renunciou a todos os direitos autorais e direitos relacionados ou vizinhos de

Reviewer: ```suggestion
  renunciou a todos os direitos autorais e direitos relacionados ou vizinhos de
```

O que esse vizinhos quer dizer?

Author: In the spanish translation we use: direitos relacionados ou conexos. Is it a good option for the Portuguese translation?

Reviewer 2: Eu fiz uma pesquisa rápida e verifiquei que "direitos conexos" é mais comum de ser encontrado em português do que "direitos relacionados". Para este último, não encontrei nada.

Reviewer 2: Eu fiz uma pesquisa rápida e verifiquei que "direitos conexos" é mais comum de ser encontrado no português do que "direitos relacionados".

>   renunciou a todos os direitos autorais e direitos relacionados ou vizinhos de

Author: ```suggestion
  renunciou a todos os direitos autorais e direitos conexos de
```
